registration/controllers/root.py

```diff
@@ -33,8 +35,14 @@ def admin(self, **kw):
     @validate(get_form(), error_handler=index)
     def submit(self, *args, **kw):
         hooks.notify('registration.before_registration', args=(kw,))
+        hooks.notify('registration.before_registration_params',
```

Comment:
This is pointless, you can get `tg.request.params` in `registration.before_registration` as `tg.request` is a global object.